### ISSUE TITLE
PayPal Payment: Email Admin Order Receipt is missing [[Order.ShippingMethod]] Token value

### ISSUE THREAD

**Issue by coretoco** [HotcakesCommerce/hotcakes-commerce-core#277]
## Describe the bug
Whenever a client pays with paypal express, the shipping method token is rendered empty in the admin order receipt. The same token is correctly rendered in the order new receipt

## Software Versions
- DNN:  08.00.04
- Hotcakes:  3.2.3

## To Reproduce
1. Go through checkout
2. select a shipping method
3. pay via paypal express

## Expected behavior
The shipping method should be in the admin order email

## Actual behavior
The shipping method is missing in the admin order email

## Error log
not available

## Additional context
The shop language is not english.

**Comment by hismightiness** [comment]
This may be an artifact of how the current PayPal Express integration works.  The shipping details are not known until the customer is returned to the website and fills in the final shipping details in the final portion of the checkout.  This is when the shipping method is known and the PayPal Express API is contacted again for the final price.

**Comment by coretoco** [comment]
But the Shipping Method is chosen _before_ paypal payment is made.
The customer email has the information in the token, only the admin mail is missing the method?

**Comment by hismightiness** [comment]
Yes, but the shipping method can be changed during the offsite checkout process that PayPal Express takes you through, so an e-commerce implementation cannot assume that the customer will return to the website having used the same shipping address, which is why there is a final checkout step where the shipping address and methods can still be chosen.

**Comment by coretoco** [comment]
As I understand the process _should_ work like this:
- you add products to the bag, click "Checkout with PayPal"
- HC redirects you to the PayPal site
- you sign in with the PayPal account or create a new one
- PayPal redirects you back to HC
- HC creates a new profile based on the information from PayPal and shows the last checkout page
- you review your order, select shipping method if any, press "Submit order" button
- PayPal charges the order in background, and you see 'thank you for your order' page

but in our shop, the workflow does take some shortcut:
- you add products to the bag, click "Checkout with PayPal"
- HC redirects you to the PayPal site
- you sign in with the PayPal account or create a new one
- PayPal redirects you back to HC
- PayPal charges the order in background, and you see 'thank you for your order' page

What's wrong here?
